lib/driver.js

```diff
@@ -196,13 +198,22 @@ class AndroidUiautomator2Driver extends BaseDriver {
       this.opts.adbPort = this.opts.adbPort || DEFAULT_ADB_PORT;
 
       await this.startUiAutomator2Session();
+      if (!caps.unitTests) {
```

Comment:
it would be better not to have test conditionals in production code. instead, we could simply stub out `fillDeviceDetails` to do nothing during unit tests